tests/test_filters.py

```diff
@@ -12,6 +12,8 @@ class TestDatetimeFilters(TestCaseBaseframe):
     def setUp(self):
         super(TestDatetimeFilters, self).setUp()
         self.now = utcnow()
+        self.date = date(2020, 1, 1)
+        self.datetime = datetime(2020, 1, 1, 0, 0)
```

Comment:
Add `self.time` for tests.

Reply:
1. Add a datetime with a UTC timezone. You currently have only a naive datetime
2. Add a datetime with a non-UTC timezone and render it with `usertz=False`